src/components/layout/MunicipalityLayout.tsx

```diff
@@ -21,17 +19,21 @@ import siteText from 'locale';
 import { WithChildren } from 'types';
 
 import useMediaQuery from 'utils/useMediaQuery';
-import { Municipal } from 'types/data';
 
 import municipalities from 'data/gemeente_veiligheidsregio.json';
+import { IMunicipalityData } from 'static-props/municipality-data';
 
 export default MunicipalityLayout;
 
 export type TMunicipality = typeof municipalities;
```

Comment:
I know this was already part of the repo, but now that I look at it again, this is probably nicer:
```suggestion
export type TMunicipalities = typeof municipalities;
export type TMunicipality = TMunicipalities[number];
```